packages/typescript-typings/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "timestamp": 1529761548,
```

Comment:
Remove the timestamp please. This is automatically added when this package version is published to NPM.